src/main/java/io/gravitee/reporter/file/config/FileReporterConfiguration.java

```diff
@@ -47,4 +59,59 @@ public Type getOutputType() {
     public long getFlushInterval() {
         return flushInterval;
     }
+
+    public Rules getRules(MetricsType type) {
+        Rules rules = new Rules();
+
+        if (environment.containsProperty(FILE_REPORTER_PREFIX + type.getType() + ".rename")) {
+            rules.setRenameFields(getMapProperties(FILE_REPORTER_PREFIX + type.getType() + "rename"));
```

Comment:
`environment.containsProperty(FILE_REPORTER_PREFIX + type.getType() + ".rename")` returns null, leading to a nullpointerexception in `FieldPropertyFilter#include()`

imho, we should use the same mechanism as in `EnvironmentUtils.getPropertiesStartingWith` to create a `EnvironmentUtils.containsPropertiesStartingWith`